install-deps.sh

```diff
@@ -546,4 +546,7 @@ if $for_make_check; then
     type git > /dev/null || (echo "Dashboard uses git to pull dependencies." ; false)
 fi
 
+source src/script/pypackages.sh
+install-pip-packages
```

Comment:
The code immediately before this will try to install all the `*requirements*.txt` files in the Ceph codebase, so we should think about a way to exclude this one from that. Or is there any reason not to?